grails-app/migrations/0.9.x/changelog-2024-05-08-1110-rename-picker-to-pickedBy-column.groovy

```diff
@@ -0,0 +1,46 @@
+databaseChangeLog = {
+
+    changeSet(author: "anadolny", id: "080520241110-0", objectQuotingStrategy: "LEGACY") {
+        preConditions(onError: "HALT", onFail: "MARK_RAN", onSqlOutput: "IGNORE") {
+            and {
+                columnExists(columnName: "picker_id", tableName: "picklist_item")
+                not {
+                    columnExists(columnName: "picked_by_id", tableName: "picklist_item")
+                }
+            }
+        }
+
+        renameColumn(
```

Comment:
@jmiranda as it was discussed on the tech huddle channel, if this is related to MySQL 5.6, let's leave it as is. However, if we'd like to be safe here, there might be a 4th option (easier than 2nd and 3rd) - we could move out the drop foreign key changeset to a separate changelog file and run it before this rename (to be sure it will be run first).